src/librustc_data_structures/sync.rs

```diff
@@ -280,21 +280,22 @@ cfg_if! {
 
         #[macro_export]
         macro_rules! parallel {
```

Comment:
Could you please add a comment why the first block gets special treatment?

Reply:
Added.